Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -770,6 +770,13 @@ def route_list_values() -> Response:
             request_args=request_args
         )
 
+        if demisto.getIntegrationContext() == {} and params.get('on_demand'):
+            values = 'You are running in On-Demand mode - please run !eis-update command to initialize the ' \
+                     'export process'
+
+        elif values == '':
```

Comment:
same here

Reply:
this is used if you are not in the first situation - "on demand mode and empty integration context" and you still got no results - that means the integration returned no results.